contracts/integrations/test/coordinator/client_test.ts

```diff
@@ -19,10 +19,9 @@ const coordinatorEndpoint = 'http://localhost:';
 const DEFAULT_PROTOCOL_FEE_MULTIPLIER = new BigNumber(150000);
 
 // tslint:disable:custom-no-magic-numbers
-blockchainTests('Coordinator Client', env => {
+blockchainTests.skip('Coordinator Client', env => {
```

Comment:
Yeah, these are broken. The `before()` step throws some error message about wanting an SQL package installed.

Reply:
@mzhu25 @xianny was this resolved or is there an Asana task that was backlogged?